app/code/Magento/Sales/Model/ResourceModel/Report/Bestsellers/Collection.php

```diff
@@ -56,6 +56,18 @@ public function __construct(
         parent::__construct($entityFactory, $logger, $fetchStrategy, $eventManager, $resource, $connection);
     }
 
+    /**
+     * Set rating limit
+     *
+     * @param int $ratingLimit
+     * @return $this
+     */
+    public function setRatingLimit($ratingLimit)
```

Comment:
Can we add strict types to newly added method?
According to the [Technical Guidelines](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html#1-basic-programming-principles) it will be nice to add strict types declaration  for this file.